sympy/integrals/tests/test_integrals.py

```diff
@@ -2050,7 +2050,7 @@ def test_sqrt_quadratic():
                   ((2*d*sqrt(a + b*x) + 2*e*(-a*sqrt(a + b*x) + (a + b*x)**(S(3)/2)/3)/b)/b, Ne(b, 0)),
                   ((d*x + e*x**2/2)/sqrt(a), True))
 
-    assert integrate((3*x**3-x**2+2*x-4)/sqrt(x**2-3*x+2)) == \
+    assert integrate((3*x**3-x**2+2*x-4)/sqrt(x**2-3*x+2)) == I am running a few minutes late; my previous meeting is running over.
```

Comment:
What is this supposed to mean. `\` allows assert equality check with the following line

Reply:
Hey I had tried to remove that, some problem with my ide ig. lemme try again

Reply:
fixed i think, pls check

Reply:
To address the failing check
You will need to add yourself to the `mailmap` file as described here to get rid of failing authors test:
https://github.com/sympy/sympy/wiki/Development-workflow#add-your-name-and-email-address-to-the-mailmap-file